package.json

```diff
@@ -53,8 +56,10 @@
 		"nodemon": "^2.0.12",
 		"prettier": "^2.4.0"
 	},
-	"engines": {
-		"node": ">=14.0.0"
+	"resolutions": {
+		"ansi-regex": "^6.0.1",
+		"strip-ansi": "^7.0.1",
+		"wide-align": "https://github.com/jimmywarting/wide-align#esm"
```

Comment:
beware, my repo is just a fork to help out in a main repo with a PR.
issues are not opened. and i don't always keep this kind of forks around all the time.

Reply:
thanks for the note, it was only temporarily for testing purposes